pkg/v1/tkg/test/tkgctl/shared/tkr_cc_misc.go

```diff
@@ -146,7 +147,7 @@ func ValidateTKRsRelatedObjectsAvailability(ctx context.Context, mcProxy *framew
 		tkr := &runv1.TanzuKubernetesRelease{}
 		err := crClient.Get(ctx, client.ObjectKey{Name: expectedTKRs[i]}, tkr)
 		Expect(err).To(BeNil(), fmt.Sprintf("failed to get TKR :%s", expectedTKRs[i]))
-		cbt := getClusterBootstrapTemplate(ctx, crClient, tkr.Name)
+		cbt := getClusterBootstrapTemplate(ctx, crClient, tkr.Name, constants.TKGNamespace)
```

Comment:
I am preserving the behavior that was before. Previously, the call to getClusterBootstrapTemplate() would result in using constants.TKGNamespace here:
https://github.com/vmware-tanzu/tanzu-framework/blob/main/pkg/v1/tkg/test/tkgctl/shared/package.go#L388

Now, we are explicitly passing the same constants.TKGNamespace from the caller because we have modified the shared function to handle tkgs as well. So, I want to keep the behavior the same as before for ValidateTKRsRelatedObjectsAvailability()